openspec/changes/readme-star-conversion-01/proposal.md

```
@@ -0,0 +1,92 @@
+  proves less clear or creates maintenance burden
+
+
+---
+
+## Source Tracking
+
+<!-- source_repo: nold-ai/specfact-cli -->
+- **GitHub Issue**: #481
+- **Issue URL**: <https://github.com/nold-ai/specfact-cli/issues/481>
+- **Last Synced Status**: proposed
+- **Sanitized**: false
```

markdownlint-cli2 (0.22.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)